src/main/kotlin/org/rust/ide/presentation/Utils.kt

```diff
@@ -58,7 +58,7 @@ fun getPresentationForStructure(psi: RsElement): ItemPresentation {
     }
     val icon = psi.getIcon(Iconable.ICON_FLAG_VISIBILITY)
     val textAttributes = when {
-        psi is RsDocAndAttributeOwner && !psi.isEnabledByCfgSelf -> RsColor.CFG_DISABLED_CODE.textAttributesKey
+        psi is RsDocAndAttributeOwner && !psi.existsAfterExpansionSelf -> RsColor.CFG_DISABLED_CODE.textAttributesKey
```

Comment:
Not sure that we should use `existsAfterExpansionSelf` here

Reply:
Ok, I'm not sure how items under attr proc macros should be shown in structure view, but it's sounds reasonable that we should not highlight them as `CFG_DISABLED_CODE`